### ISSUE TITLE
Dropdown align left or right

### ISSUE THREAD

**Issue by alannsiqueira** [Dogfalo/materialize#512]
The dropdown don't align right, only left.
I have one solution for this problem, 

Note: I'm using ionic Framework and the materializecss in version 0.95.0;

New option are add:
                    $('.dropdown-button').dropdown({
                        alignLeft: false
                    }); 

and the code on materialize.js :
 $.fn.dropdown = function (options) {
    var defaults = {
      inDuration: 300,
      outDuration: 225,
      constrain_width: true, // Constrains width of dropdown to the activator
      hover: true,
      alignLeft: true,
    }

    options = $.extend(defaults, options);
    this.each(function(){
    var origin = $(this);

    // Dropdown menu
    var activates = $("#"+ origin.attr('data-activates'));

    // Move Dropdown menu to body. This allows for absolute positioning to work
    if ( !(activates.parent().is($('body'))) ) {
      activates.detach();
      $('body').append(activates);
    }

    /*
      Helper function to position and resize dropdown.
      Used in hover and click handler.
    */
    function placeDropdown() {
      var dropdownRealHeight = activates.height();
      var leftPos =  origin.offset().left;
      if (options.constrain_width === true) {
        activates.css('width', origin.outerWidth());
      }
      
      if (!options.alignLeft){
         leftPos =  origin.offset().left-activates.outerWidth()+origin.outerWidth();
      }
      
      if (elementOrParentIsFixed(origin[0])) {
        activates.css({
          display: 'block',
          position: 'fixed',
          height: 0,
          top: origin.offset().top - $(window).scrollTop(),
          left: leftPos
        });
      }
      else {
        activates.css({
          display: 'block',
          top: origin.offset().top,
          left: leftPos,
          height: 0
        });
      }

**Comment by Dogfalo** [comment]
Added in 679115764057fb56ebc736e326c3f16b9c4fbbf9
